package.json

```diff
@@ -43,7 +51,8 @@
   ],
   "dependencies": {
     "browserslist-config-monday": "^1.0.6",
-    "classnames": "^2.3.2"
+    "classnames": "^2.3.2",
+    "@storybook/types": "^7.5.0"
```

Comment:
It should be a dependency right? We can't put it into devDeps?

Reply:
I think it should be a dev dependency